src/types/prerecordedTranscriptionResponse.ts

```diff
@@ -1,12 +1,59 @@
 import { Metadata } from "./metadata";
 import { Channel } from "./channel";
 import { Utterance } from "./utterance";
+import { secondsToTimestamp } from "../helpers";
 
-export type PrerecordedTranscriptionResponse = {
+export class PrerecordedTranscriptionResponse {
   request_id?: string;
   metadata?: Metadata;
   results?: {
     channels: Array<Channel>;
     utterances?: Array<Utterance>;
   };
-};
+
+  /**
+   * Converts the transcription to the WebVTT format
+   * @remarks In order to translate the transcription to WebVTT, the utterances
```

Comment:
I'm not a TS-versed person, but will this fail gracefully if the user does not have `utterances`? It might be nice to provide that as a clear note so users can troubleshoot.

Reply:
Yeah I think that's important. Either in the output file, or just throw an error back to the user if there is no utterances array.

Reply:
Updated to throw an error